src/systems/physics/Physics.cc

```diff
@@ -2197,6 +2199,22 @@ void PhysicsPrivate::RemovePhysicsEntities(const EntityComponentManager &_ecm)
 void PhysicsPrivate::UpdatePhysics(EntityComponentManager &_ecm)
 {
   GZ_PROFILE("PhysicsPrivate::UpdatePhysics");
+  // Gravity state
+  _ecm.Each<components::Gravity>(
+      [&](const Entity & _entity, const components::Gravity *_gravity)
+      {
+      int worldCount = engine->GetWorldCount();
```

Comment:
Is there a reason we need this?
I presume the entityWorldMap should returna valid world. That being said Im not an expert in our physics systems.

Reply:
It crashed Gazebo if you issue the service command too soon

Reply:
Yup thanks for giving the idea it did work infact